hail/src/test/scala/is/hail/expr/ir/EmitStreamSuite.scala

```diff
@@ -14,10 +14,10 @@ import org.testng.annotations.Test
 
 class EmitStreamSuite extends HailSuite {
 
-  private def compileStream[F >: Null : TypeInfo, A](
+  private def compileStream[F >: Null : TypeInfo, AA](
     streamIR: IR,
     inputTypes: Seq[PType]
-  )(call: (F, Region, A) => Long): A => IndexedSeq[Any] = {
+  )(call: (F, Region, AA) => Long): AA => IndexedSeq[Any] = {
```

Comment:
Why change the As to AAs?

Reply:
oops! Was getting a compilation warning=>error, will revert.